csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -10,16 +10,55 @@ namespace Platform.IO
 {
     public static class ConsoleHelpers
     {
+        /// <summary>
+        /// <para>Displays in the console appealing to press any key for continue</para>
+        /// <para>Выводит в консоли призыв нажать любую клавишу для продолжения</para>
+        /// </summary>
+        /// <returns>
+        /// <para>Returns true if a user pressed any key.</para>
+        /// <para>Возвращает истину если пользователь нажал любую клавишу</para>
+        /// </returns>
```

Comment:
This method returns nothing. 
Learn more about [the void keyword](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/void#:~:text=You%20use%20void%20as%20the,doesn't%20return%20a%20value.&text=You%20can%20also%20use%20void,pointer%20to%20an%20unknown%20type.&text=You%20cannot%20use%20void%20as%20the%20type%20of%20a%20variable.).